ui/src/app/views/homepage/component.css

```diff
@@ -377,7 +377,7 @@
 }
 
 .image {
-  height: 136px;
+  max-height: 136px;
```

Comment:
Is this related to the svg changes?

Reply:
Yup, to prevent png's from getting stretched.